core/src/avm2/globals/flash/filters/BlurFilter.as

```diff
@@ -0,0 +1,17 @@
+﻿package flash.filters {
+	public final class BlurFilter extends BitmapFilter {
+		public var blurX: Number;
+		public var blurY: Number;
+		public var quality: int;
+
+		public function BlurFilter(blurX: Number = 4.0, blurY: Number = 4.0, quality: int = 1) {
+			this.blurX = blurX;
+			this.blurY = blurY;
+			this.quality = qualty;
+		}
+
+		override public function clone(): BitmapFilter {
+			return new BitmapFilter(this.blurX, this.blurY, this.quality);
```

Comment:
Have you meant `new BlurFilter`?

Reply:
Fixed - it seems that `asc.jar` doesn't attempt to resolve the constructor call.